src/rulesets/oas/__tests__/oas3-server-trailing-slash.ts

```diff
@@ -25,6 +25,19 @@ describe('oas3-server-trailing-slash', () => {
     expect(results.length).toEqual(0);
   });
 
+  test('validate a correct object with default value', async () => {
```

Comment:
```suggestion
  test('validate a correct object with a value containing trailing slash', async () => {
```

Reply:
Or actually, screw that one. You took it from the spec, which is good.
Feel free to ignore my comment.